src/test/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProjectTest.java

```diff
@@ -267,4 +271,16 @@ public DescriptorImpl() {
         r.assertLogContains("branch=feature2", b2);
     }
 
+    @Test public void getProjectClassBySymbol(){
```

Comment:
This is not really testing anything not apparent from source code. Unless there is some user-visible effect of the change that can be asserted, may as well just skip the test coverage.

Reply:
removed.